### PR TITLE
Document MQTT schemas

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Improve documentation of MQTT schemas for MQTT light and MQTT vacuum

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/light.mqtt.markdown [hunk 1]
```diff
@@ -8,7 +8,8 @@ ha_iot_class: Configurable
 ha_release: 0.8
 ---
 
-The `mqtt` light platform lets you control your MQTT enabled lights through one of the supported message schemas.
+The `mqtt` light platform lets you control your MQTT enabled lights through one of the supported message schemas, `default`, `json` or `template`.
+Schema is chosen by setting the `schema` configuration parameter.
```
Comment (frenck) [inline source/_integrations/light.mqtt.markdown]: Isn't it a bit weird to refer to a configuration option in the introduction text?
A new user hasn't seen that part yet... Maybe just remove this line?
```suggestion
The schema is chosen by setting the `schema` configuration parameter.
```

File: source/_integrations/light.mqtt.markdown [hunk 2]
```diff
@@ -501,6 +507,11 @@ rgb:
   required: false
   type: boolean
   default: false
+schema:
+  description: The schema to use. Must be `json` to select the json schema".
```
Comment (frenck) [inline source/_integrations/light.mqtt.markdown]:
```suggestion
  description: The schema to use. Must be `json` to select the JSON schema".
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @emontnemery!

Reviewed the PR and left 2 small comments/suggestions. Could you take a look? Thanks! 👍

File: source/_integrations/light.mqtt.markdown (hunk 1 above)
Comment (emontnemery) [reply source/_integrations/light.mqtt.markdown]: Yeah, line removed.

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery! 👍

**Pull request merged**